hazelcast-hibernate/hazelcast-hibernate4/src/test/java/com/hazelcast/hibernate/entity/AnnotatedEntity.java

```diff
@@ -0,0 +1,46 @@
+package com.hazelcast.hibernate.entity;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.NaturalId;
+import org.hibernate.annotations.NaturalIdCache;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@NaturalIdCache
+@Entity
+@Table( name = "ANNOTATED_ENTITIES" )
+public class AnnotatedEntity {
+    private Long id;
+
+    private String title;
+
+    public AnnotatedEntity() {
+    }
+
+    public AnnotatedEntity(String title) {
+        this.title = title;
+    }
+
+    @Id
+    @GeneratedValue(generator="increment")
+    @GenericGenerator(name="increment", strategy = "increment")
+    public Long getId() {
+        return id;
+    }
+
+    private void setId(Long id) {
+        this.id = id;
+    }
+
+    @NaturalId(mutable = true)
```

Comment:
can't we put this somewhere on top? At my first check, I just did'nt see `@NaturalId` annotation...